.github/workflows/tests.yml

```
@@ -4,24 +4,29 @@ on: [push]
 
 jobs:
   build-and-test:
-    
-    runs-on: ubuntu-latest
-
+	strategy:
```

actionlint

could not parse as YAML: yaml: line 7: found character that cannot start any token

(syntax-check)